lib/vdc/CurvilinearGrid.cpp

```diff
@@ -312,7 +312,9 @@ CurvilinearGrid::ConstCoordItrCG::ConstCoordItrCG(
 	_xCoordItr = rhs._xCoordItr;
 	_yCoordItr = rhs._yCoordItr;
 	_zCoordItr = rhs._zCoordItr;
-	_terrainFollowing = rhs._terrainFollowing;
+	if (rhs._terrainFollowing) {
+		_terrainFollowing = rhs._terrainFollowing;
+	}
```

Comment:
This seems unrelated

Reply:
It is. I'm not sure how this snuck in here, but it does fix a memory access error.